### ISSUE TITLE
Merge PBP?

### ISSUE THREAD

**Issue by DocMAX** [RupertAvery/PSXPackager#1]
As this tool obviously can't create multi-disk PBP files, any idea how to merge them as command line batch?
Edit: Oh this is no command line tool. So never mind.

**Comment by RupertAvery** [comment]
Hey, I'm adding support for multi-disc PBPs. The way I've figured is to use the M3U file format. 

- Your input file will be an .m3u containing the paths to the cue or bin. 
- If it is a multi-track cue it will merge it before processing it further.

I don't know how to test multi-track cues in multi-disc PBP.

Lunar Silver Star Story has 2 discs and has multi-tracks for audio.  I was able to load Disc 2 from the generated PBP in epsxe and play the intro (interestingly, it also has the playable intro sequence game, before it asks to switch to disc 1.
